colpali_engine/models/florence2/colflor/processing_colflor.py

```diff
@@ -0,0 +1,89 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+from PIL import Image
+from transformers import BatchFeature
+
+from colpali_engine.utils.processing_utils import BaseVisualRetrieverProcessor
+
+from .processing_florence2 import Florence2Processor

```

Comment:
Can we make the two imports absolute? Not super important, but i'd be great to be ISO with the rest of the codebase.

```suggestion
from colpali_engine.models.florence2.colflor.processing_florence2 import Florence2Processor
from colpali_engine.utils.processing_utils import BaseVisualRetrieverProcessor
```